src/cmd-buildprep

```diff
@@ -12,10 +12,23 @@ import requests
 import botocore
 import boto3
 import shutil
+from tenacity import retry, stop_after_delay, stop_after_attempt, retry_if_exception_type
 
 sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
 from cmdlib import load_json, rm_allow_noent, Builds  # noqa: E402
 
+retry_stop = (stop_after_delay(10) | stop_after_attempt(5))
+retry_requests_exception = (retry_if_exception_type(requests.Timeout) | \
+                            retry_if_exception_type(requests.ReadTimeout) | \
+                            retry_if_exception_type(requests.ConnectTimeout) | \
+                            retry_if_exception_type(requests.ConnectionError))
+retry_s3_exception = (retry_if_exception_type(botocore.exceptions.ConnectionClosedError) |
```

Comment:
Would be good to share this across the two commands; could stick it in `cmdlib.py` I guess, or make a new `s3utils.py`?

Reply:
Moved these to `cmdlib.py`, I don't think its worth creating `s3utils.py` just yet, its being used in 2 files only